AGENTS.md

```
@@ -0,0 +1,200 @@
+
+### Database Migrations
+```bash
+make alembic-revision message="Description"  # Create migration
+make alembic-upgrade                         # Apply migrations
+make alembic-downgrade                       # Rollback one version
+```
+
+## Architecture
+
+### Monorepo Structure
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)